tests/providers/test_base_provider.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+import os
+from pathlib import Path
+from typing import Iterator
+
+import pytest
+
+from commitizen.config.base_config import BaseConfig
+from commitizen.exceptions import VersionProviderUnknown
+from commitizen.providers import get_provider
+from commitizen.providers.commitizen_provider import CommitizenProvider
+
+
+@pytest.fixture
+def chdir(tmp_path: Path) -> Iterator[Path]:
+    cwd = Path()
+    os.chdir(tmp_path)
+    yield tmp_path
+    os.chdir(cwd)
```

Comment:
As this has been moved to `tests/providers/contest.py`, I think we're good to remove it.

```suggestion
```